### PR TITLE
Makefile fix

### PR DESCRIPTION
**Pull request by charleenfei**
exports environmental variables and set correct prototool version in Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,18 +14,25 @@ help: ## Show this help message.
...
 
 install: ## Install dependencies required to generate bindings & documentation
-install: install_dep vendorinstall
-	
+install: env version install_dep vendorinstall
+
+env:
+	export PROTOTOOL_VERSION=0.4.0 export PROTOTOOL_BIN=~/bin/0.4.0/prototool
+
+version:
```
Comment (vedhavyas) [inline Makefile]: lets call this install_prototool ?

File: Makefile [hunk 2]
```diff
@@ -14,18 +14,25 @@ help: ## Show this help message.
 	@egrep '^(.+)\:\ ##\ (.+)' ${MAKEFILE_LIST} | column -t -c 2 -s ':#'
 
 install: ## Install dependencies required to generate bindings & documentation
-install: install_dep vendorinstall
-	
+install: env version install_dep vendorinstall
+
+env:
```
Comment (vedhavyas) [inline Makefile]: can we move this exports to install_prototool?

File: Makefile [hunk 3]
```diff
@@ -14,18 +14,25 @@ help: ## Show this help message.
...
+	export PROTOTOOL_VERSION=0.4.0 export PROTOTOOL_BIN=~/bin/0.4.0/prototool
+
+version:
+	mkdir -p ~/bin
+	PROTOTOOL_VERSION=$$PROTOTOOL_VERSION ./scripts/install_prototool.sh
```
Comment (vedhavyas) [inline Makefile]:
```suggestion
	PROTOTOOL_VERSION=$(PROTOTOOL_VERSION) ./scripts/install_prototool.sh
```

File: Makefile (hunk 2 above)
Comment (lucasvo) [reply Makefile]: Setting environment variables only affects the context of the makefile, right? Isn't there another way to do this in the makefile without the need to call export?

Comment (charleenfei) [reply Makefile]: resolving this conversation because I have made edits to the Makefile.

File: Makefile (hunk 3 above)
Comment (charleenfei) [reply Makefile]: made the suggested changes, thanks!

**Pull request merged**